functions/Get-DbaSqlBuildReference.ps1

```diff
@@ -0,0 +1,210 @@
+Function Get-DbaSqlBuildReference
+{
+<#
+.SYNOPSIS
+Returns SQL Server Build infos on a SQL instance
+
+.DESCRIPTION
+Returns info about the specific build of a SQL instance, including the SP, the CU and the reference KB, wherever possible.
+It also includes End Of Support dates as specified on Microsoft Lifecycle Policy
+
+.PARAMETER SqlInstance
+The SQL Server that you're connecting to.
+
+.PARAMETER Credential
+Credential object used to connect to the SQL Server as a different user.
+
+.PARAMETER Build
+Instead of connecting to a real instance, pass a string identifying the build to get the info back.
+
+.PARAMETER Silent
+Replaces user friendly yellow warnings with bloody red exceptions of doom!
+
+.NOTES
+Author: niphlod
+Tags: SqlBuild
+
+dbatools PowerShell module (https://dbatools.io, clemaire@gmail.com)
+Copyright (C) 2016 Chrissy LeMaire
+This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+You should have received a copy of the GNU General Public License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+.LINK
+https://dbatools.io/Get-DbaSqlBuildReference
+
+.EXAMPLE
+Get-DbaSqlBuildReference -SqlInstance sqlserver2014a
+
+Returns SqlInstance, SPLevel, CULevel, KBLevel, SupportedUntil for server sqlserver2014a
+
+.EXAMPLE
+$cred = Get-Credential sqladmin
+Get-DbaSqlBuildReference -SqlInstance sqlserver2014a -SqlCredential $cred
+
+Does the same as above but logs in as SQL user "sqladmin"
+
+.EXAMPLE
+Get-DbaSqlBuildReference -Build "12.00.4502"
+
+Returns information about a build identified by  "12.00.4502" (which is SQL 2014 with SP1 and CU11)
+
```

Comment:
Please include examples:
- piping sqlinstance(s) to the function
- submitting multiple sqlinstances